projects.css

```diff
@@ -0,0 +1,205 @@
+.parent-container {
+  display: grid;
+  grid-template-rows: auto auto;
+  row-gap: 12px;
+  padding-right: 16px;
+  padding-left: 16px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 40px;
+}
+
+h2 {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-bottom: 20px;
+}
+
+.nav-elements {
+  display: flex;
+}
+
+.nav-elements a {
+  text-decoration: none;
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin-right: 12px;
+}
+
+.counter {
+  background-color: #c1c7d0;
+  margin-right: 12px;
+}
+
+.nav-elements .current-nav {
+  color: #344563;
+}
+
+p {
+  font-family: 'poppins', Arial, Helvetica, sans-serif;
+  font-weight: 400px;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.languages {
+  display: flex;
+  align-items: center;
+  flex: start;
+}
+
+.language-button {
+  margin-right: 10px;
+  color: #6070ff;
+  border: none;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-weight: 500px;
+  font-size: 12px;
+  line-height: 16px;
+  padding: 4px 12px;
+  border-radius: 9px;
+  margin-bottom: 20px;
+}
+
+li {
+  list-style: none;
+  margin: 0;
+}
+
+ul .languages {
+  padding: 0 0 0 0;
+}
+
+.see-project-button,
+.get-my-resume-button {
+  padding: 12px;
+  color: #396df2;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  border: 1px solid #396df2;
+  margin-bottom: 10px;
+  border-radius: 10px;
+  background-color: white;
+}
+
+section {
+  background-color: white;
+  border-radius: 5%;
+  padding-top: 20px;
+}
+
+.about-myself {
+  padding-left: 24px;
+  padding-right: 24px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  margin-bottom: 220px;
+}
```

Comment:
- [ ] The about myself section should be aligned using flexbox. Please add the flex display and use it to style this section since this is a project requirement.
 - [ ] This section on the template has a border radius on the top right corner, transitioning from the last section to this one due to the different background colors. Please add that.